src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/Options.cs

```diff
@@ -365,8 +386,6 @@ private void LoadReferencedAssemblies(IList<string> referenceArgs)
                     {
                         if (Tool.IsFatal(e))
                             throw;
-
-                        throw new ToolOptionException(SR.Format(SR.ErrCouldNotLoadReferenceAssemblyAt, path), e);
                     }
```

Comment:
We should emit a warning message saying we were unable to load this reference assembly.